### PR TITLE
feat: add ngneat/overview to package.json by schematics

### PR DESCRIPTION
**Pull request by stefanoww**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #30

## What is the new behavior?

@ngneat/overview is added to the package.json by the ng add schematic.

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: schematics/src/ng-add/index.ts [hunk 1]
```diff
@@ -54,15 +56,24 @@ export function addImportsToModuleDeclaration(options: SchemaOptions, imports: s
   };
 }
 
+function addModuleToPackageJson() {
+  return (host: Tree, context: SchematicContext) => {
+    addPackageToPackageJson(host, "dependencies", "@ngneat/overview", "1.0.0");
```
Comment (stefanoww) [inline schematics/src/ng-add/index.ts:61]: I have a problem with hard coded version "1.0.0" here. @NetanelBasal Do you have an idea how we can get the latest version to install?

Comment (NetanelBasal) [reply schematics/src/ng-add/index.ts:61]: Use `^1.0.0`

Comment (NetanelBasal) [reply schematics/src/ng-add/index.ts:61]: Or use https://www.npmjs.com/package/latest-version

Comment (stefanoww) [reply schematics/src/ng-add/index.ts]: Ok, currently I have used ^1.0.0

**Comment by NetanelBasal** [comment]
Thanks. Can you please update the README to use ng add?

**Comment by stefanoww** [comment]
> Thanks. Can you please update the README to use ng add?

Ok no problem, but now I have found an bug, because now running `ng add @ngeneat/helicopter` adds `overview`, but does not add `helipopper` package. What's more I have created tests to the schematics but currently they do not run on the `ng test` command so I need some more time to figure that things out.

**Comment by stefanoww** [comment]
@NetanelBasal done, take a look if README is fine.

**Pull request merged**